future-apps/payroll/contracts/Payroll.sol

```diff
@@ -82,15 +82,15 @@ contract Payroll is EtherTokenConstant, IForwarder, IsContract, AragonApp {
     mapping(address => bool) internal allowedTokens;
     address[] internal allowedTokensArray;
 
-    event AddAllowedToken(address token);
+    event AddAllowedToken(address indexed token);
     event SetEmployeeSalary(uint256 indexed employeeId, uint256 denominationSalary);
     event AddEmployeeBonus(uint256 indexed employeeId, uint256 amount);
     event AddEmployeeReimbursement(uint256 indexed employeeId, uint256 amount);
     event AddEmployeeAccruedSalary(uint256 indexed employeeId, uint256 amount);
-    event TerminateEmployee(uint256 indexed employeeId, address indexed accountAddress, uint64 endDate);
-    event ChangeAddressByEmployee(uint256 indexed employeeId, address indexed oldAddress, address indexed newAddress);
-    event DetermineAllocation(uint256 indexed employeeId, address indexed employee);
-    event SendPayment(address indexed employee, address indexed token, uint256 amount, uint128 exchangeRate, string paymentReference);
+    event TerminateEmployee(uint256 indexed employeeId, uint64 endDate);
+    event ChangeAddressByEmployee(uint256 indexed employeeId, address indexed newAccountAddress, address indexed oldAccountAddress);
+    event DetermineAllocation(uint256 indexed employeeId);
+    event SendPayment(uint256 indexed employeeId, address indexed employeeAccountAddress, address indexed token, uint256 amount, uint128 exchangeRate, string paymentReference);
```

Comment:
Generally it seems unnecessary to emit the employee's address as it's retrievable through `employeeId`.

However, I've kept it in `SendPayment` to allow us to search for payroll payments to particular addresses in the future.